tensorboard/components/tf_paginated_view/tf-paginated-view.html

```diff
@@ -237,13 +237,16 @@
     },
     observers: ['_clampActiveIndex(items)'],
 
-    _validateLimit(limit) {
-      if (limit !== Math.floor(limit)) {
-        throw new Error(`limit must be an integer, but got: ${limit}`);
-      }
-      if (limit <= 0) {
-        throw new Error(`limit must be positive, but got: ${limit}`);
-      }
+    attached() {
+      this._limitListener = () => {
+        this.set('_limit', store.getLimit());
+      };
+      store.addLimitListener(this._limitListener);
+      this._limitListener();
+    },
+
+    detached() {
+      store.removeLimitListener(this._limitListener);
```

Comment:
Nice performance improvement for people away from the tab.

Reply:
I believe that it's also necessary for correctness: otherwise, paginated views that get unmounted will try to `set` polymer properties on instances that no longer exist, which should be undefined behavior.

Reply:
Ah k! Makes sense.